INSS.EIIR.DataSync.Infrastructure/Sink/XML/XMLSink.cs

```diff
@@ -49,11 +49,11 @@ public XMLSink(XMLSinkOptions options, IExtractRepository extractRepository, IEx
 
             if (string.IsNullOrEmpty(_blobContainerName))
             {
-                throw new Exception("XML Sink options - No StorageName check XmlContainer setting in configuration");
+                throw new XmlSinkIirException("XML Sink options - No StorageName check XmlContainer setting in configuration", null);
```

Comment:
instead of null , actual InnerExcpetion would be helpful

Reply:
There is actually no InnerException here to throw which is why it is null